pure-pso/values.yaml

```diff
@@ -27,6 +27,9 @@ storagetopology:
   enable: false
   strictTopology: false
 
+# this option is to enable/disable the pso smartagent feature
```

Comment:
this comment may cause some confusion as what smart agent is. we may need detailed documents to describe the container and its use cases.